extension/payment/mockedRules/DE.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @param {SDKContext} context
+ * @param {PaymentMethodInput} input
+ * @returns {Promise<PaymentMethod[]>|PaymentMethod[]}
+ */
+module.exports = async (context, input) => {
+  if (!input.checkout.paymentAddress || input.checkout.paymentAddress.countryCode !== 'NL') {
```

Comment:
Why `paymentAddress` ? what is it? I will rename back to `billingAddress`

Reply:
What will happen if  `if (!input.checkout.paymentAddress` will be truthy.

I guess `paymentAddress.countryCode !== 'NL') {` is wrong here. Should be `DE` and checked 1 more time. Otherwise rules will apply to any country code